pupil-auth/config.js

```diff
@@ -0,0 +1,86 @@
+'use strict'
+require('dotenv').config()
+const os = require('os')
+const toBool = require('to-bool')
+
+const twoMinutesInMilliseconds = 120000
+const oneMinuteInMilliseconds = 60000
+
+const getEnvironment = () => {
+  return process.env.ENVIRONMENT_NAME || 'Local-Dev'
+}
+
+module.exports = {
+  AZURE_STORAGE_CONNECTION_STRING: process.env.AZURE_STORAGE_CONNECTION_STRING,
+  GOOGLE_TRACKING_ID: process.env.GOOGLE_TRACKING_ID,
+  MONGO_CONNECTION_STRING: process.env.MONGO_CONNECTION_STRING || 'mongodb://localhost/mtc',
+  NCA_TOOLS_AUTH_URL: process.env.NCA_TOOLS_AUTH_URL,
+  PORT: process.env.PORT || '3002',
+  PUPIL_APP_URL: process.env.PUPIL_APP_URL,
+  QUESTION_TIME_LIMIT: 5,
+  RESTART_MAX_ATTEMPTS: 2,
+  SESSION_SECRET: process.env.NODE_ENV === 'production' ? process.env.SESSION_SECRET : 'anti tamper for dev',
+  TIME_BETWEEN_QUESTIONS: 2,
```

Comment:
It doesn't show questions does it?

Reply:
there are quite a lot of env vars that can be removed.  Dan may need some guidance on this